Sources/Networking/Core/Upload/UploadTask+State.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  UploadTask+State.swift
+//  
+//
+//  Created by Tony Ngo on 12.06.2023.
+//
+
+import Foundation
+
+extension UploadTask {
```

Comment:
Should be this extension also public?

Reply:
Making this extension wouldn't change anything. Although, I can mark it as public and remove the public access level from the `UploadTask.State` on line 12.